content/how-to/upgrade-from-webpack-1.md

```diff
@@ -182,3 +182,60 @@ To keep compatiblity with old loaders, this information can be passed via plugin
 +   })
   ]
 ```
+
+
+#### Code Splitting with ES6
+
+The ES6 Loader spec defines `System.import` as method to load ES6 Modules dynamically on runtime.
+
+Webpack threads `System.import` as splitpoint and puts the requested module in a separate chunk.
+
+`System.import` takes the module name as argument and returns a Promise.
+
+``` js
+function onClick() {
+	System.import("./module").then(module => {
+		module.default;
```

Comment:
Haha this is so funny because as someone who wasn't around during webpack 2, I've never used the `require.ensure` call so I'm not sure what the equivalent would be. Let me fetch an example